hfinger/hfinger_core.py

```diff
@@ -191,6 +191,47 @@ def get_content_type(hdr):
     return hdr_coded + ",".join(ret)
 
 
+def get_cache_control_value(hdr):
+    hdr_value_table = CACHECONT
```

Comment:
Is the temp variable needed?
`HDRL` seems to be similar and it's used directly

Reply:
Addressed in https://github.com/CERT-Polska/hfinger/pull/6/commits/9b02056573571d5372639ef4b5bb54302942ac66. Please check :-)